patches/chromium/fix_when_building_with_enable_basic_printing_false.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Milan Burda <milan.burda@gmail.com>
+Date: Sat, 28 Sep 2019 10:39:58 +0200
+Subject: fix: when building with enable_basic_printing=false
+
+
```

Comment:
Patches must include justification. Also, it seems like this patch should join the other printing mega-patch?

Reply:
merged with `printing.patch`